plugins/spammer/spammer.go

```diff
@@ -37,12 +38,45 @@ func doSpam(shutdownSignal <-chan struct{}) {
 	}
 
 	if !tangle.IsNodeSyncedWithThreshold() {
+		// log.Infof("worker idle because: !tangle.IsNodeSyncedWithThreshold()")
+		randomSleep()
 		return
 	}
 
+	if !tangle.IsNodeSynced() {
+		// log.Infof("worker idle because: !tangle.IsNodeSynced()")
+		randomSleep()
```

Comment:
Why do we even need these sleeps at all? Just abort the `doSpam` without the sleeps, because it is triggered by a ticker anyway.